### DISCUSSION TITLE
Symfony 5.3.6 with PHP 7.4.22 or PHP 8.0.9 ?

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#42745] [Q&A] [answered]
https://developers.ibexa.co/blog/benchmarks-php-7.4-8.0-jit-opcache-preloading-symfony
This site say that php 7.4 is better than php 8 for Symfony 5.
Is it TRUE ?
Which PHP version is better for symfony 5.3.6 ?

**Comment by Nek-** [comment, marked as answer]
I think you can say performances are basically the same since it really depends on what you are doing inside your code. But you can improve your code performance with some tricks like type-hinting everything in functions arguments. But this is not really useful and is most likely a loss of time.

On the other hand, if you have workers, then PHP 8 with JIT may be [faster by far](https://www.youtube.com/watch?v=dWH65pmnsrI).

[Most benchmark](https://www.phoronix.com/scan.php?page=news_item&px=PHP-8.0-Performance-Bench-2020) considers PHP 8 faster. But it's not always the case indeed. And even though the difference is not a good reason to keep PHP 7.4 because the difference is not significant.
